modules/pensions/spec/lib/pensions/monitor_spec.rb

```diff
@@ -17,136 +17,56 @@
 
     describe '#track_show404' do
       it 'logs a not found error' do
-        log = '21P-527EZ submission not found'
-        payload = {
-          confirmation_number: claim.confirmation_number,
-          user_uuid: current_user.uuid,
-          message: monitor_error.message
-        }
-
-        expect(Rails.logger).to receive(:error).with(log, payload)
-
+        expect_any_instance_of(Logging::Monitor).to receive(:track_request)
```

Comment:
should change these to `expect(monitor).to receive(:track_request).with` the correct log and payload

Reply:
@wayne-weibel I was having issues with the correct payload when it came to the call_location but I'll try it again

Reply:
can use 'anything' in the test to allow any call location.  not sure if you could spoof 'caller_locations' though